client/js/render.js

```diff
@@ -64,11 +64,7 @@ function appendMessage(container, chanId, chanType, msg) {
 	}
 
 	// Always create a condensed container
-	const newCondensed = buildChatMessage(chanId, {
-		type: "condensed",
-		time: msg.time,
-		previews: []
-	});
+	const newCondensed = $(templates.msg_condensed({time: msg.time}));
```

Comment:
There's no need to build a complete message since it uses a separate template so we can just optimize it.